src/components/ble/NimbleController.cpp

```diff
@@ -71,7 +71,7 @@ void NimbleController::Init() {
   ASSERT(res == 0);
   res = ble_svc_gap_device_name_set(deviceName);
   ASSERT(res == 0);
-  rc = ble_svc_gap_device_appearance_set(0xC4);
+  rc = ble_svc_gap_device_appearance_set(0xC2);
```

Comment:
Not sure if I cloned it right, but the variable `rc` does not exist in that function. I had to change it to `res` variable whic is used to check return values from BLE functions.
```
/home/martin/Desktop/dev/watches/InfiniTime/src/components/ble/NimbleController.cpp: In member function 'void Pinetime::Controllers::NimbleController::Init()':
/home/martin/Desktop/dev/watches/InfiniTime/src/components/ble/NimbleController.cpp:74:3: error: 'rc' was not declared in this scope; did you mean 'erfc'?
   74 |   rc = ble_svc_gap_device_appearance_set(0xC2);
      |   ^~
```

Reply:
The device name and appearance are not set properly in Init as discussed in the comments.